mist-api/src/routes/genome_by_taxid/get.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
```

Comment:
Exact replica of `routes/genomes/get.js`. Remove this file.

Reply:
well, if we will work on `parseCriteriaForMany`, then we should just reject this branch altogether and rework the code to support simple query matching. I am going to close it.